src/pages/migration/redeploy-file-storage.md

```diff
@@ -87,14 +91,14 @@ _Note:_ References to source ACD or source namespace are referring to your exist
 
 1. Redeploy your cartridges to the new ACD instance so they are persisted to the new storage volume.
 
-  - See [Deploying and updating the ACD-provided cartridges](/usage/getting-started/#deploying-and-updating-the-acd-provided-cartridges) for deploying the provided cartridges.
-  - See [Cartridge deployment](/usage/customizing/#cartridge-deployment) for deploying your custom cartridges.
+    - See [Deploying and updating the ACD-provided cartridges](/usage/getting-started/#deploying-and-updating-the-acd-provided-cartridges) for deploying the provided cartridges.
+    - See [Cartridge deployment](/usage/customizing/#cartridge-deployment) for deploying your custom cartridges.
 
 1. Make OAuth proxy updates for secure access to the target namespace, if configured to use this.
 
-  For ACD deployments (not the Configuration Editor), create a serviceview role in target namespace and grant all service accounts in the proxy namespace access to it. Follow the steps 7 and 8 under [Manage Access](/security/manage-access/).
+    - For ACD deployments (not the Configuration Editor), create a serviceview role in the target namespace and grant all service accounts in the proxy namespace access to it. Follow the steps 7 and 8 under [Manage Access](/security/manage-access/).
```

Comment:
Suggestion on the 4 migration pages that have "(not the Configuration Editor)", would be to say "(not the Configuration Editor deployment"), or "(not the [Configuration Editor](https://merative.github.io/acd-containers/config-editor/overview) deployment)".  
On 3 of the 4 migration pages, Configuration Editor is not mentioned anywhere previously on the page, so adding a little more context might be helpful.